doc/user-guide/Push-notifications.md

```diff
@@ -72,7 +74,7 @@ Also, it's important to note, that the first node type on the `plugins` list, wi
 
 ##### [mod_push_service_mongoosepush][]
 
-This module acts as a bridge between [mod_pubsub][] that receives notifications from [mod_push][] and passes those to [MongoosePush][] which sends them to _FCM_ and/or _APNS_. To enable this module type in your configuration:
+This module acts as a bridge between [mod_pubsub][] that receives notifications from [mod_event_pusher_push][] and passes those to [MongoosePush][] which sends them to _FCM_ and/or _APNS_. To enable this module type in your configuration:
```

Comment:
This module acts as a bridge between [mod_pubsub][] that receives notifications from [mod_event_pusher_push][] and passes them to [MongoosePush][] which sends them to _FCM_ and/or _APNS_. To enable this module type in your configuration: